CONTRIBUTING.md

```diff
@@ -26,4 +26,21 @@ The simplest procedure is to run `npm run watch` which executes
 directory and will recompile any changed files. If you prefer to compile
 them manually, use `npm run compile` which will run `ucompiler go` for you.
 
+## Releasing
+
+Project members with push access to the repository also have the permissions
+needed to release a new version.  If there have been changes to the project and
+the team decides it is time for a new release, the process is to:
+
+1. Update `CHANGELOG.md` with the planned version number and a short bulleted
+list of major changes.  Include pull request numbers if applicable.
+1. Commit the changelog changes to master.
+1. Publish a new version with `apm publish <version>` where the version number
```

Comment:
This seems to be saying you should write the command like `apm publish vX.Y.Z`, you can simply run `apm publish {major|minor|patch}` to have `apm` automatically perform the appropriate incrementation for you. (eg: `apm publish patch`)